cub/test/CMakeLists.txt

```diff
@@ -129,6 +129,17 @@ function(cub_is_catch2_test result_var test_src)
   endif()
 endfunction()
 
+## cub_cdp_by_lid
+#
+# If given launcher id corresponds to a CDP launcher, set `out_var` to 1.
+function(cub_cdp_by_lid out_var launcher_id)
```

Comment:
Suggestion: rename this function to `_cub_launcher_requires_rdc`.

Reply:
The intention of the underscore was to distinguish between "heavy" functions that might be used repeatedly across multiple files as a sort of API vs. helper functions that just simplify code flow in our implementations (like these one-off functions that don't do anything interesting except map one value into another to make life a little easier).

This was more important for thrust, which exposes some functions in the cmake package (`thrust_create_target`) that are actual user APIs. My goal was to have the more useful functions appear in IDE autocompletion while the helpers wouldn't clutter up the list of completions. It doesn't have any actual impact on correctness, though, it was just a convention.

In this suggestion, the new name was more important with the underscore. `launcher_requires_rdc` makes it clear that this is used solely to map launcher values to RDC flags.

Reply:
Thank you for elaborating on this! I've changed the function name per recommendation. The convention doesn't seem intuitive. If I'm not missing anything, it's not documented. Maybe it's worth adding cmake developer overview, describing it.